lib/drivers/medtronic600/medtronic600Simulator.js

```diff
@@ -216,8 +253,6 @@ class Medtronic600Simulator {
               .with_rate(this.currBasal.suppressed.rate)
               .with_scheduleName(this.currBasal.suppressed.scheduleName)
               .with_duration(Date.parse(event.time) - Date.parse(this.currBasal.basalEndTime))
-              .with_conversionOffset(this.currBasal.conversionOffset)
```

Comment:
Same here. If you're using `fillInUTCInfo`, you should only use `with_deviceTime`, not `with_time`